### PR TITLE
Pretty print quotes

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala [hunk 1]
```diff
@@ -51,4 +51,16 @@ class DecompilerPrinter(_ctx: Context) extends RefinedPrinter(_ctx) {
       }
     }
   }
+
+  override protected def toTextCore[T >: Untyped](tree: Tree[T]): Text = {
```
Comment (allanrenucci) [inline compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala]: Why is not part of `RefinedPrinter`?

Comment (nicolasstucki) [reply compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala]: No, we do not want this representation when debugging.

File: compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala [hunk 2]
```diff
@@ -51,4 +51,16 @@ class DecompilerPrinter(_ctx: Context) extends RefinedPrinter(_ctx) {
...
+    import untpd.{modsDeco => _, _}
+    tree match {
+      case TypeApply(fun, args) =>
```
Comment (allanrenucci) [inline compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala]: I think we should have a protected `def` for each type of tree and override the bits we are interested in.,

Comment (nicolasstucki) [reply compiler/src/dotty/tools/dotc/printing/DecompilerPrinter.scala]: Added it

**Pull request merged**